Formula/qscintilla2.rb

```diff
@@ -59,25 +64,33 @@ def install
     if build.with?("python") || build.with?("python3")
       cd "Python" do
         Language::Python.each_python(build) do |python, version|
+          pyargs = ["--apidir=#{prefix}/qsci"]
+          pyargs << "--spec=#{spec}"
+          pyargs << "--qsci-sipdir=#{share}/sip"
+          if build.with?("qt5")
+            pyargs << "--pyqt=PyQt5"
+            pyargs << "--destdir=#{lib}/python#{version}/site-packages/PyQt5"
+            pyargs << "--pyqt-sipdir=#{Formula["pyqt5"].opt_prefix}/share/sip/Qt5"
```

Comment:
You're looking for `opt_share` here rather than `opt_prefix/share`. Is it actually installing anything there or is this purely a reference?